torch/csrc/distributed/rpc/python_rpc_handler.cpp

```diff
@@ -1,13 +1,30 @@
 #include <torch/csrc/distributed/rpc/python_rpc_handler.h>
 #include <torch/csrc/distributed/rpc/rpc_agent.h>
 #include <torch/csrc/jit/pybind_utils.h>
+#include <torch/csrc/utils/python_compat.h>
 
 namespace torch {
 namespace distributed {
 namespace rpc {
 
 namespace {
 
+// A macro that grabs the GIL, profiling the acquisition time. The average GIL
+// acquisition time will be recorded in RpcAgent's getMetrics().
+#define PROFILE_GIL_SCOPED_ACQUIRE                                       \
+  std::chrono::time_point<std::chrono::high_resolution_clock> startTime; \
```

Comment:
Lint message

Reply:
This change is just moving that existing # define from the .h to the .cpp, to reduce scope.

The lint complaint is about using # define in general, nothing specific that's addressable without somewhat bigger changes

Reply:
Actually looks like a bug in the linter since we're not actually declaring a constant here but an entire codeblock.